portfolio/src/main/java/com/google/sps/data/TaskGroup.java

```diff
@@ -0,0 +1,65 @@
+package com.google.sps.data;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Iterator;
+import java.util.List;
+import java.util.PriorityQueue;
+
+/**
+ * This class holds a List of tasks and includes functions to sort
+ * the List and retrieve elements.
+ */
+public class TaskGroup {
```

Comment:
1. Consider renaming to TaskQueue, because it is conceptually a queue.
2. Can we have some unit test for this class?

Reply:
Done.